Cargo.lock

```diff
@@ -104,7 +104,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0944d18a9a37691b87733b39c9360c9950af9aa5f97e2455bc108d8eb64fc1c1"
 dependencies = [
  "bitvec",
- "blake2s_simd",
+ "blake2s_simd 0.5.11",
```

Comment:
Why does bellman need 0.5.11 rather than 1.0.0?

Reply:
Because it hasn't been updated yet. It's not used in its public API, so we can bump it in a point release; I just didn't have time before the RC.